src/legacy/status_im/ui/screens/signing/views.cljs

```diff
@@ -106,24 +102,25 @@
             (when-not in-progress? {:on-press #(re-frame/dispatch [:signing.ui/cancel-is-pressed])}))
      (i18n/label :t/cancel)]]])
 
-(views/defview keycard-pin-view
+(defn keycard-pin-view
   []
-  (views/letsubs [pin           [:keycard/pin]
-                  small-screen? [:dimensions/small-screen?]
-                  error-label   [:keycard/pin-error-label]
-                  enter-step    [:keycard/pin-enter-step]
-                  status        [:keycard/pin-status]
-                  retry-counter [:keycard/retry-counter]]
-    (let [enter-step    (or enter-step :sign)
-          margin-bottom (if platform/ios? 40 0)]
-      [react/view {:margin-bottom margin-bottom}
-       [pin.views/pin-view
-        {:pin           pin
-         :retry-counter retry-counter
-         :step          enter-step
-         :small-screen? small-screen?
-         :status        status
-         :error-label   error-label}]])))
+  [react/view]
+  #_(views/letsubs [pin           [:keycard/pin]
```

Comment:
I usually don't see much of a point in keeping large sections of commented out code because we have a decent git history, with commits leading to PRs, so it's relatively easy to go back in time and grab old code in case we need it.

Would you mind just deleting commented out code in this PR?

Reply:
yeah i agree

Reply:
done